### PR TITLE
Fix yaf stubs

### PR DESCRIPTION
**Pull request by mbakirov**
This PR contain fixes for YAF stubs against https://github.com/laruence/yaf/ repo

### CODE REVIEW

**Review by isfedorov — CHANGES_REQUESTED** [review]

File: yaf/yaf.php [hunk 1]
```diff
@@ -1295,11 +1351,16 @@ abstract public function execute();
...
 
+    /**
+     * @return string
+     */
     public function getControllerName() {}
 }
 
 /**
  * @link https://secure.php.net/manual/en/class.yaf-config-abstract.php
+ * @implements \Iterator<mixed,mixed>
```
Comment (isfedorov) [inline yaf/yaf.php:1362]: Please remove <mixed, mixed> since it doesn't make sense

**Review by isfedorov — CHANGES_REQUESTED** [review]

**Comment by mbakirov** [comment]
@isfedorov thank you for your comments! I've fixed them all

**Comment by mbakirov** [comment]
@isfedorov may I ask your review please?

**Review by isfedorov — CHANGES_REQUESTED** [review]
@mbakirov  Sorry for the long review. Looks like it's only a couple of final notes left. Could you please take a look at them and we will merge PR after that.

**Comment by mbakirov** [comment]
@isfedorov I have removed all `@implements` phpdoc comments

**Pull request merged**